include/vicon_estimator.h

```diff
@@ -184,7 +202,7 @@ class RotationalEstimatorResults {
   Eigen::Vector3d rate_old_;
   Eigen::Quaterniond orientation_estimate_;
   Eigen::Vector3d rate_estimate_;
-  bool measurement_outlier_flag_;
+  bool measurement_rotational_outlier_flag_;
```

Comment:
Same as above. This is in the class "RotationalEstimatorResults", respecification of rotational is unnecessary.